lib/index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 
 var libvirt = require('bindings')('libvirt'),
-    Promise = require("bluebird");
+    Promise = require("bluebird"),
+    EventEmitter = require('events').EventEmitter;
+
+// extend prototype
+function inherits(target, source) {
+    for (var k in source.prototype) {
+        target.prototype[k] = source.prototype[k];
+    }
+}
```

Comment:
Why not use https://nodejs.org/docs/latest/api/util.html#util_util_inherits_constructor_superconstructor instead?